themes/red-theme/lms/templates/header.html

```diff
@@ -1,6 +1,6 @@
 ## mako
 <%namespace name='static' file='static_content.html'/>
-<%namespace file='main.html' import="login_query, stanford_theme_enabled"/>
```

Comment:
I don't think anyone is actually *using* the red-theme, but if we remove stanford_theme_enabled from this location, does that mean any templates which reference it (such as course_about.html) will raise an exception if this particular theme is being applied and no template override is provided?  If so we may want to hold off on this particular change until all references to stanford_theme_enabled are removed from the templates.